composer.json

```diff
@@ -23,7 +23,8 @@
 		"php": ">=5.3.0",
 		"wikibase/data-model": "~6.0|~5.0|~4.2",
```

Comment:
yes, this should be changed to 6.2 when it is released.
This will also solved the PHP fatal you're quoting above. Although you've merged the PR adding foreign ids to data model it still needs to be tagged.
To test this PR locally you could temporarily hack composer json to use master (or rather the branch of https://github.com/wmde/WikibaseDataModel/pull/679)